### DISCUSSION TITLE
a few questions about microstream

### DISCUSSION THREAD

**Discussion by guykatz** [microstream-one/microstream#499] [General]
Hi all;
bumped into this lib lately
is this library reactive? wierdly i cant find info in my regular outlets. vert.x doesnt seem to have a plugin, quarkus has almost zero info (except a hackathon winning project which is outdated), wildfly integration - almost no info.
its a bit strange i must say. I was expecting to get easy info on the web...
i know micronaut and hellidon have integrations but they are currently not a part of my stack.
thanks!

**Comment by rdebusscher** [comment]
Hi,

No, MicroStream is not reactive. It has blocking parts when you save the object Graph entries (to protect the single shared instances).

We have prepared a Quarkus integration that will be official released with the new version 8.0 soon (The EA release nots can be found [here](https://github.com/microstream-one/microstream/releases/tag/08.00.00-MS-EA1)).

We have a CDI integration that requires MicroProfile Config that can be used on WildFly. See [here](https://www.infoq.com/news/2022/05/microstream-7-0/), [here](https://dzone.com/articles/ultra-fast-microservices-part-2-when-microstream-m), and [here](https://docs.microstream.one/manual/misc/integrations/cdi.html) for more info.

There is indeed no vert.x plugin as we are not reactive and thus it makes not much sense.

**Reply by guykatz** [reply]
thanks a lot for the very informative anwer!!!
BTW. i have scheduled a talk with your reps from the web interface but no one showed up...
